core/src/main/java/tc/oc/pgm/cycle/CycleMatchModule.java

```diff
@@ -47,7 +47,7 @@ public void startCountdown(@Nullable Duration duration) {
   public void onPartyChange(PlayerPartyChangeEvent event) {
     if (event.wasParticipating()
         && match.isRunning()
-        && match.getParticipants().size() < PGM.get().getConfiguration().getMinimumPlayers()) {
+        && match.getParticipants().isEmpty()) {
```

Comment:
This would end matches even if `min-players` is set to 0, which is often done in debug/test server scenarios so that your matches don't end on you if you rejoin